src/fu-tool.c

```diff
@@ -4937,6 +4964,13 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Removes devices to watch for future emulation"),
 			      fu_util_emulation_untag);
+	fu_util_cmd_array_add(cmd_array,
+			      "emulation-load",
+			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+			      _("FILENAME"),
+			      /* TRANSLATORS: command description */
+			      _("Load device emulation data"),
+			      fu_util_emulation_load);
```

Comment:
it's just a few steps away from a command like `device-emulate` to run the emulation json files.  Maybe that's worth adding too?